Coverage plugin fails when fails to load covered module

```
Traceback (most recent call last):
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/cli.py", line 413, in main
    environments=options.environments, tasks=arguments)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/reactor.py", line 128, in build
    return self.build_execution_plan(tasks, execution_plan)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/reactor.py", line 166, in build_execution_plan
    reactor=self)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 351, in execute_execution_plan
    summaries.append(self.execute_task(task, **keyword_arguments))
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 295, in execute_task
    if self.execute_action(action, keyword_arguments):
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 340, in execute_action
    action.execute(arguments)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/execution.py", line 100, in execute
    self.callable(*arguments)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/karellen_pyb_plugin/__init__.py", line 154, in run_coverage
    "run_unit_tests", shortest_plan=True)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/plugins/python/coverage_plugin.py", line 79, in run_coverage
    shortest_plan))
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/utils.py", line 323, in fork_process
    raise_exception(result[1], result[2])
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/excp_util_3.py", line 27, in raise_exception
    raise ex.with_traceback(tb)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/utils.py", line 305, in instrumented_target
    send_value = (target(*args, **kwargs), None, None)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/plugins/python/coverage_plugin.py", line 125, in do_coverage
    allow_non_imported_modules)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/pybuilder/plugins/python/coverage_plugin.py", line 167, in _list_all_covered_modules
    module = __import__(module_name)
  File "/home/arcivanov/Documents/src/karellen/karellen-kombu-ext/src/main/python/karellen/kombu/transport/django/models.py", line 11, in <module>
    class Queue(models.Model):
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/django/db/models/base.py", line 105, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/django/apps/registry.py", line 237, in get_containing_app_config
    self.check_apps_ready()
  File "/home/arcivanov/.pyenv/versions/3.5.2/envs/karellen-venv/lib/python3.5/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
------------------------------------------------------------
BUILD FAILED - Apps aren't loaded yet.
------------------------------------------------------------
```

--- a/src/main/python/pybuilder/plugins/python/coverage_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/coverage_plugin.py
@@ -166,8 +166,10 @@ def _list_all_covered_modules(logger, module_names, modules_exceptions, allow_no
             try:
                 module = __import__(module_name)
             except SyntaxError as e:
-                logger.warn("Coverage for module '%s' cannot be established - syntax error: %s",
-                            module_name, e)
+                logger.warn("Coverage for module '%s' cannot be established - syntax error: %s", module_name, e)
+                continue
+            except Exception as e:
+                logger.warn("Coverage for module '%s' cannot be established - module failed: %s", module_name, e)
                 continue
 
         if module not in modules and hasattr(module, "__file__"):

